common-core-open/src/main/java/edu/isi/nlp/files/KeyValueSources.java

```diff
@@ -92,4 +93,26 @@ private KeyValueSources() {
     }
     return new ZipKeyValueSource(zipFile, ret.build());
   }
+
+  @Nonnull
+  public static ImmutableKeyValueSource<Symbol, ByteSource> fromVistaUtilsZipKeyValueSource(
+      final ZipFile zipFile) {
+    // read "__keys" file
```

Comment:
That is reads `__keys` is clear from the code; can you make the comment tell what is special about the `__keys` file?

Reply:
yes, sorry about that. these comments were leftover from my draft